packages/web/src/components/search/addons/Mic.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import types from '@appbaseio/reactivecore/lib/utils/types';
 import MicImage from '../../../styles/MicImage';
+import { getComponent, hasCustomRenderer } from '../../../utils';
 
 const STATUS = {
 	initial: 'INITIAL',
-	started: 'STARTED',
+	stopped: 'STOPPED',
 	allowed: 'ALLOWED',
 	deined: 'DENIED',
```

Comment:
```suggestion
	denied: 'DENIED',
```

Reply:
This would also go on other places where we are accessing this key

Reply:
@jyash97 I didn't get you here.

Reply:
We will need to change from `STATUS.deined` to `STATUS.denied`